db/app.js

```diff
@@ -1,13 +1,18 @@
 const express = require("express")
 const {selectAllCategories} = require("./controllers/categories.controller")
-const {notFound} = require("./error-handling/categories.error")
+const {notFound, badRequest, malformedEndpoint} = require("./error-handling/errors")
+const {selectReview} = require("./controllers/reviews.controller")
+
 
 const app = express();
 
 app.use(express.json());
 
 app.get("/api/categories", selectAllCategories);
+app.get("/api/reviews/:review_id", selectReview)
 
-app.use("/api/*", notFound);
+app.use("/api/*", malformedEndpoint);
+app.use("/api/reviews/:review_id", notFound)
```

Comment:
app.use() can take the error handler function, it doesn't need a path/endpoint. Please check the 'Error handling' section of the notes.